bootstrap/utils.go

```diff
@@ -0,0 +1,378 @@
+package bootstrap
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/ed25519"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"github.com/dgrijalva/jwt-go"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/square/go-jose.v2"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"stash.kopano.io/kc/konnect/signing"
+	"strings"
+)
+
+func parseJSONWebKey(jsonBytes []byte) (*jose.JSONWebKey, error) {
+	k := &jose.JSONWebKey{}
+	if err := k.UnmarshalJSON(jsonBytes); err != nil {
+		return nil, err
+	}
+	return k, nil
+}
+
+func LoadSignerFromFile(fn string) (string, crypto.Signer, error) {
```

Comment:
Why is this exported while some of the other functions are not. In any case, exported functions need a comment.